lib/linguist/generated.rb

```diff
@@ -588,7 +588,8 @@ def generated_graphql_relay?
     def generated_gamemakerstudio?
       return false unless ['.yy', '.yyp'].include? extname
       return false unless lines.count > 3
-      return lines[2].match(/\"modelName\"\:\s*\"GM/)
+      return lines[2].match(/\"modelName\"\:\s*\"GM/) ||
+             lines[0] =~ /^\d\.\d\.\d.+\|\{/
```

Comment:
Is this (version?) number always at the very beginning of the file?

Reply:
Yes, currently i've only seen `1.0.0`, but I think this is the most important part for the IDE - look at it as a header, specifying the version, so the IDE knows how to read the file - in that way it makes sense to be the very first piece of text.

You can find a couple this way; https://www.google.com/search?q=site:github.com%20options/main/inherited/options_main.inherited.yy

Reply:
Hm, I was going to suggest that we make that a `\A`, but I missed that you are already matching on the first line only. So all good for me :+1: